Unix/server/server.c

```diff
@@ -894,6 +895,13 @@ static void GetConfigFileOptions()
             if (SetPathFromNickname(key, value) != 0)
                 err(ZT("SetPathFromNickname() failed"));
         }
+        else if (strcasecmp(key, "NtlmCredsFile") == 0)
+        {
+            if (value)
+            {
+                s_opts.ntlmCredFile = PAL_Strdup(value);
```

Comment:
The value is pre-parsed according to the same rules as every other value in the file.

I'm not clear what case we are trying to solve.